src/components/package/show/package-show.js

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-
+import _ from 'lodash';
```

Comment:
We should be more specific here so we don't import the entire `lodash` library if we can avoid it. If you need deepCopy, then you should be able to do `import deepCopy from 'lodash/deepCopy'`. If this doesn't work, let me know.